salt/_states/caasp_file.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+
+import salt.utils.files
+
+
+def managed(name, **kwargs):
+    '''
+    Manage a given file, this function allows for a file to be downloaded from
+    the salt master and potentially run through a templating system.
+
+    This is a wrapper on the standard :py:func:`file.managed <salt.states.file.managed>`
+    state where we can specify a `work_dir` that will be used for creating the temporary
+    file, as the standard version creates a temporary file in the same directory as `name`,
+    and that can lead to some problems with programs/daemons that are watching that
+    directory (like the kubelet with `/etc/kubernetes/manifests`).
+
+    work_dir
+        A directory for creating temporary files.
+
+    For a full list of arguments see :py:func:`file.managed <salt.states.file.managed>`
+    '''
+    def debug(s):
```

Comment:
`_debug` if it is a private func

Reply:
It is not a private function but a local function (as a local variable). So we do not have to follow the `_` convention used for limiting scope because it is impossible to access this function from the outside.